dvc/scm/tree.py

```diff
@@ -57,9 +66,13 @@ def walk(self, top, topdown=True):
         def onerror(e):
             raise e
 
-        for root, dirs, files in os.walk(
-            top, topdown=topdown, onerror=onerror
+        for root, dirs, files in dvc_walk(
+            top,
+            topdown=topdown,
+            onerror=onerror,
+            ignore_file_handler=ignore_file_handler,
         ):
-            if topdown:
-                dirs[:] = [i for i in dirs if i not in (".git", ".hg", ".dvc")]
             yield os.path.normpath(root), dirs, files
+
+    def get_tree_root(self):
+        return self.repo_root
```

Comment:
but `repo_root` is already public, why add a getter on top?

Reply:
To unify method of accessing Tree root, this method looks different for example in GitTree. However we are not using it there so maybe there is no point in doing that yet.

Reply:
I see. So maybe just wrap it into a property?